everviz/plugins/summary_plot/summary_plot.py

```diff
@@ -1,6 +1,6 @@
 from pathlib import Path
 from uuid import uuid4
-from collections import deque
+from itertools import cycle
```

Comment:
The changes in this file look fine, but can you put them in a different commit, doesn´t quite belong in the same as objectives?

Reply:
I can do that, are we ok with putting it in the same pull request? It is just a minor change for consistency.

Reply:
Yes, the same PR is ok by me.